lib/tasks/evm_application.rb

```diff
@@ -110,4 +110,13 @@ def self.update_stop
     File.write(tempfile, " ")
     stop
   end
+
+  def self.set_region_file(region_file, new_region)
+    old_region = region_file.exist? ? region_file.read.to_i : nil
+
+    return if new_region == old_region
+
+    _log.info("Changing REGION file from [#{old_region}] to [#{new_region}]. Restart to use the new region.")
+    region_file.write(new_region)
```

Comment:
do we even need to write the REGION file anymore?  I thought @kbrock got rid of that requirement.